colossus-metrics/src/main/scala/colossus/metrics/Metric.scala

```diff
@@ -15,7 +13,7 @@ object OpenTsdbFormatter extends MetricFormatter[String] {
 
```

Comment:
These changes came about when I found a fun issue when concatting a MetricAddress.Root with "/".  When I found out what the culprit was, it looks like this class needed a bit of an update.  Everything now should work identically:
construction and appending can be invoked with strings that don't have a preceding "/".  When invoking "/" with a String that contained "/" in them it didn't properly break out the components.  I fixed that by having the /(str:String) just create a MetricAddress and invoke the other "/" function, so that behavior is identical in all cases.